generate-manifest

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import logging
+import os
+import shutil
+import subprocess
+import sys
+import traceback
+import urllib.parse
+import xml.etree.ElementTree as ET
+
+from xml.dom import minidom
+
+logging.basicConfig(format="%(levelname)s: %(message)s")
+logger = logging.getLogger(__name__)
+
+BASE_PATH = os.path.realpath(os.path.dirname(__file__))
+KOLIBRI_CONTENT_EXTENSION_BASE_ID = "org.learningequality.Kolibri.Content"
+MANIFEST_TEMPLATE_PATH = "{}/manifest.json.in".format(BASE_PATH)
+MANIFEST = None
+KOLIBRI_HOME_PATH = "{}/kolibri_home".format(BASE_PATH)
+CONTENT_JSON_PATH = "{}/content/content.json".format(KOLIBRI_HOME_PATH)
+METAINFO_DEFAULTS = {
+    "metadata_license": "CC0-1.0",
+    "url_homepage": "https://learningequality.org/",
+    "project_group": "Learning Equality",
+    "project_license": "MIT",
```

Comment:
The code that generates these extensions may be MIT licensed but the content within them is not. I wonder whether it would be better to omit `project_license` by default, or require it to be specified for the specific content? I guess it may be rather hard to extract it from the content itself.

Reply:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-project_license says:

>  Although the project_license tag is not mandatory, it is highly recommended to include it.